panther_diagnostics/src/system_status_node.cpp

```diff
@@ -0,0 +1,174 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "panther_diagnostics/system_status_node.hpp"
+
+#include <chrono>
+#include <exception>
+#include <filesystem>
+#include <fstream>
+
+#include "diagnostic_updater/diagnostic_updater.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+#include "cppuprofile/uprofile.h"
```

Comment:
Accordingly to Husarion code-style guide

```suggestion
#include "cppuprofile/uprofile.h"

#include "diagnostic_updater/diagnostic_updater.hpp"
#include "rclcpp/rclcpp.hpp"

```